docs/install/aws-install.md

```diff
@@ -45,36 +45,36 @@ The key components deployed on your EKS cluster are:
 
 **Parameters**
 
-2. **EKS cluster name** (m)&mdash;the name of EKS cluster created. The EKS cluster is used to run the MLRun services. For example, `https://jupyter.<eks_cluster_name>.<route53_domain_name>`
+2. **EKS cluster name** (m)&mdash;the name of EKS cluster created. The EKS cluster is used to run the MLRun services. For example: "John-1".
 
 **VPC network Configuration**
 
 3. **Number of Availability Zones** (m)&mdash;number of availability zones. The default is set to 3. Choose from the dropdown to change the number. The minimum is 2.
 4. **Availability zones** (m)&mdash;select a zone from the dropdown. The list is based on the region of the instance. The number of zones must match the number of zones Number of Availability Zones.
-5. **Allowed external access CIDR** (m)&mdash;range of IP address allowed to access the cluster. Addresses that are not in this range are will not be able to access the cluster.
+5. **Allowed external access CIDR** (m)&mdash;range of IP address allowed to access the cluster. Addresses that are not in this range will not be able to access the cluster. Contact your IT manager/network administrator if you are not sure what to fill here.
 
 **Amazon EKS configuration**
 
 6. **Additional EKS admin ARN (IAM user)** (o)&mdash;add an additional admin user to the instance. Users can be added after the stack has been created. For more information see [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
 7. **Instance type** (m)&mdash;select from the dropdown list. The default is m5.2xlarge. For size considerations see [Amazon EC2 Instance Types](https://aws.amazon.com/ec2/instance-types/).
-8. **Number of Nodes** (m)&mdash;number of nodes in the cluster. The minimum must match the number of **Availability Zones**. The number of nodes combined with the **Instance type** determines the AWS infrastructure cost. 
+8. **Maximum Number of Nodes** (m)&mdash;maximum number of nodes in the cluster. The minimum value of this parameter must match the number of **Availability Zones**. The number of nodes combined with the **Instance type** determines the AWS infrastructure cost.
 
 **Amazon EC2 configuration**
 
 9. **SSH key name** (m)&mdash;select from the stored keys in the dropdown. The list is based on the SSH keys that are in your account. For more information about SSH Keys see [Amazon EC2 key pairs and Linux instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
-10. **Provision bastion host** (m)&mdash;create a bastion host for SSH access to the Kubernetes nodes. The default is enabled.
+10. **Provision bastion host** (m)&mdash;create a bastion host for SSH access to the Kubernetes nodes. The default is enabled. This will allow ssh access to your EKS EC2 instances through a public IP.
```

Comment:
10. **Provision bastion host** (m)&mdash;create a bastion host for SSH access to the Kubernetes nodes. The default is enabled. This allows ssh access to your EKS EC2 instances through a public IP.